source/_integrations/systemmonitor.markdown

```diff
@@ -56,23 +56,36 @@ file.
 | load_1m                |                           |
 | load_5m                |                           |
 | load_15m               |                           |
-| network_in             | Interface, e.g., `eth0`   |
-| network_out            | Interface, e.g., `eth0`   |
-| throughput_network_in  | Interface, e.g., `eth0`   |
-| throughput_network_out | Interface, e.g., `eth0`   |
-| packets_in             | Interface, e.g., `eth0`   |
-| packets_out            | Interface, e.g., `eth0`   |
-| ipv4_address           | Interface, e.g., `eth0`   |
-| ipv6_address           | Interface, e.g., `eth0`   |
+| network_in             | Interface*, e.g., `eth0`   |
+| network_out            | Interface*, e.g., `eth0`   |
+| throughput_network_in  | Interface*, e.g., `eth0`   |
+| throughput_network_out | Interface*, e.g., `eth0`   |
+| packets_in             | Interface*, e.g., `eth0`   |
+| packets_out            | Interface*, e.g., `eth0`   |
+| ipv4_address           | Interface*, e.g., `eth0`   |
+| ipv6_address           | Interface*, e.g., `eth0`   |
```

Comment:
What does `*` mean in this case?

Reply:
Not sure what you mean by "key in configuration root".
Will add the column.

Reply:
If this is a required configuration options, why isn't it part of the format configuration scheme instead of attribute list level stuff?

Reply:
Is there a way via the voluptuous schema to make a an element conditionally required based on the value of another schema element? If yes, then this mandatory requirement could be enforced via the schema as well. But I couldn't find something.

The other option would be to create separate schemas for all supported sensor types and then combine them into one.

What I just added as an alternative is add a manual check during platform setup, if all required arguments are provided and if not log an error.
https://github.com/spacegaier/core/commit/3248ec81800488de391a8857297a8ad76494b5dd

Should I create a PR for that?

Reply:
Well for the documentation PR, that obviously doesn't matter that much, as the goal is to match documentation with the actual code.

However, I think it is weird to have optional attributes that are mandatory but sort of not force to be mandatory.